### ISSUE TITLE
Customisation options for Font Size in "Big Number" tiles

### ISSUE THREAD

**Issue by IrakliJani** [lightdash/lightdash#4945]
### Description

ability to customise the appearance of the "Big Number" feature by changing the font size. 

Suggested Solution:

Add a customization option within the "Big Number" tile settings that allows users to change the font size to better fit their preferences and requirements.

**Comment by jpetey75** [comment]
I don't think this is specific to big values, merging with a generic font issue
